pubsub/psclient/client.go

```diff
@@ -92,7 +92,7 @@ func newPingMsg(reqID int64) []byte {
 	return pingMsg
 }
 
-var defaultTimeout = 10 * time.Second
+const defaultTimeout = pubsub.PingInterval * 10 / 9
```

Comment:
Could be additive (e.g. `pubsub.PingInterval + 2 * time.Second`), but the timeout used by the client instance is still configurable via `psclient.Opts`.